diffmpm/mesh.py

```diff
@@ -1,43 +1,111 @@
 import jax.numpy as jnp
+from jax import vmap
+from node import Nodes
+from particle import Particles
 
-from diffmpm import element
-from diffmpm import node
-from diffmpm import particle
 
 class Mesh1D:
```

Comment:
Why this is Mesh1D while other classes don't have the dimension in their name?

Reply:
My try is to make the container classes such that for all dimensions, you can still call the same constructor, maybe with dimension as an argument. But at some point, we would want to handle things differently for different dimensions. Mesh seemed like a good place where this differentiation can be implemented. If we then want to have a consistent API for the user for Mesh like the other classes, we can create a parent class and make all these dimension specific classes as private classes. This was one of the things that I want to discuss today.